### PR TITLE
Fixes an EC2 provisioning zmq hang

### PR DESCRIPTION
**Pull request by Dannyb48**
This should fix a zmq hang I was having in Carbon when provisioning ec2 resources even though I had setup no_monitor evar and context keys.

### CODE REVIEW

File: linchpin/provision/action_plugins/ec2.py [hunk 1]
```diff
@@ -37,6 +37,78 @@ def set_dict(context, key, value):
             context.__setitem__ = set_dict
         else:
             context = zmq.Context()
+            context = zmq.Context()
```
Comment (14rcole) [inline linchpin/provision/action_plugins/ec2.py]: Duplicate lines

Comment (Dannyb48) [reply linchpin/provision/action_plugins/ec2.py]: Good catch. fixed.

**Comment by 14rcole** [comment]
@Dannyb48 can you rebase against the develop branch?

**Comment by Dannyb48** [comment]
@14rcole done

**Comment by abraverm** [comment]
[merge]

**Pull request merged**